modules/openxr/openxr_api.cpp

```diff
@@ -2739,6 +2793,19 @@ RID OpenXRAPI::get_action_rid(XrAction p_action) {
 	return RID();
 }
 
+RID OpenXRAPI::find_action(String p_name) {
+	List<RID> current;
+	action_owner.get_owned_list(&current);
+	for (int i = 0; i < current.size(); i++) {
+		Action *action = action_owner.get_or_null(current[i]);
+		if (action && action->name == p_name) {
+			return current[i];
```

Comment:
```suggestion
	for (const RID &E : current) {
		Action *action = action_owner.get_or_null(E);
		if (action && action->name == p_name) {
			return E;
```
Avoid random access to `List`